app/controllers/account/admin/categories_controller.rb

```diff
@@ -0,0 +1,94 @@
+# Encoding: utf-8
+module Account
+  module Admin
+    class CategoriesController < ApplicationController
+      before_action :admin_or_moderator?
+      before_action :status_inspector, only: [:show]
+      before_action :find_category, except: [:index, :create, :new]
+
+      def index
+        @categories = Category.all
```

Comment:
@denys281, I intentionally turned it off. Do we need it if there unlikely wouldn't be more than 10-20 categories?